src/_settings.scss

```diff
@@ -61,6 +61,9 @@ $c-gray-lightest: #f1f1f1 !default;
 $c-text-base: $c-gray-dark !default;
 $c-text-accent: $c-accent !default;
 $c-text-link: $c-text-accent !default;
+$td-text-link: underline !default;
+$td-hover-text-link: none !default;
+$fz-text-link: null !default;
```

Comment:
I see this PR merged, but somehow we are getting more and more Sass-default-variable-addicted. Don't see the point of setting useless variables globally, didn't you think of setting default values locally - just for the mixin? Wouldn't it solve your problem?

Reply:
I've considered this. But the mixin contained one var already so I didn't want it to have two different ways of setting vars. So I created these default vars. The following doesn't look good for me:
```css
@mixin some ($a: 'red'){
    color: $a; // passed by parameter
    margin: $b; // defined as default var
}
```

Reply:
Why not? `$b` is used globally, `$a` not. I'm just warning about the big amount of variables we're defining in the settings file, and setting variables useless doesn't help too much. E.g.: `$fz-text-link: null !default;` doesn't make sense for me. But if anyone else thinks this is a real problem, let's go ahead.